pkgs/applications/graphics/processing/default.nix

```diff
@@ -108,7 +139,10 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Language and IDE for electronic arts";
     homepage = "https://processing.org";
-    license = with licenses; [ gpl2Only lgpl21Only ];
+    license = with licenses; [
```

Comment:
Consider adding `sourceProvenance` here as well. Look through the tree for examples.